### PR TITLE
find multiple of two numbers

### PR DESCRIPTION
**Pull request by ra7kar**

### CODE REVIEW

File: data_structures/is_multiple.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# is multiple of a number.
+# numbers n and m, where n = mi.
+# return true if n is a multiple of m else return false.
+
+
+def is_multiple(n, m):
```
Comment (atrifex) [inline data_structures/is_multiple.py:6]: Shouldn't this return a `True` or `False`

Comment (ra7kar) [reply data_structures/is_multiple.py:6]: return value corrected as True or False.

File: data_structures/is_multiple.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    ret_val = False
+    for i in range(1, n + 1):
```
Comment (atrifex) [inline data_structures/is_multiple.py:8]: Don't need a loop to do this.

Comment (ra7kar) [reply data_structures/is_multiple.py:8]: The for loop is removed and mod operator is used to check if the numbers are multiples of each other.

**Pull request merged**